src/mrinufft/extras/gradient.py

```diff
@@ -6,7 +6,7 @@
 
 
 @with_numpy
-def cg(operator, kspace_data, x_init=None, num_iter=10, tol=1e-4):
+def cg(operator, kspace_data, x_init=None, num_iter=10, tol=1e-4, compute_loss=False):
     """
     Perform conjugate gradient (CG) optimization for image reconstruction.
 
```

Comment:
I think we can remove this if we have added its support in base.py already.

Reply:
actually lets wait to see what @paquiteau thinks of having both a function and also the operator way to access this